pkg/kubelet/cm/dra/checkpoint/state_checkpoint.go

```diff
@@ -20,55 +20,19 @@ import (
 	"fmt"
 	"sync"
 
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager"
 	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager/errors"
+	checkpointapi "k8s.io/kubernetes/pkg/kubelet/cm/dra/checkpoint/v1"
```

Comment:
can we not just alias this to `checkpoint`?  Calling `checkpointapi.New()`  is more awkward than simply calling `checkpoint.New()`.

Reply:
I can do that, but we may need to import checkpoint APIs from the `k8s.io/kubernetes/pkg/kubelet/cm/dra/checkpoint`. How would you suggest to call this import?